app/models/study_json_record/processor_v2.rb

```diff
@@ -158,6 +158,20 @@ def result_agreement_data
   end
 
   def result_contact_data
+    return unless results_section
+
+    nct_id = protocol_section.dig('identificationModule', 'nctId')
+    point_of_contact = results_section.dig('moreInfoModule', 'pointOfContact')
+    return unless point_of_contact
+   {
+      nct_id: nct_id,
+      ext: point_of_contact['phoneExt'],
+      phone: point_of_contact['phone'],
+      title: point_of_contact['title'],

```

Comment:
It looks like the title is mapped to 'name' look at the old method.

Reply:
Thanks for reviewing, error is fixed.